doc/integration/InputFormat.md

```diff
@@ -0,0 +1,31 @@
+# Lumen data input format
+
+Lumen currently supports import of data in CSV format either from a local file or from a web address (URL). Lumen will import your data and, to it's best capacity, guess the correct datatype of the CSV columns when importing it, thus saving you time in the data cleaning step.
+
+## File format
+
+A UTF-8 encoded file or file stream containing values separated by a comma `,`.
+
+The first row can include column headers or not. If the file is imported without column headers, these will get names generated automatically (`Column 1, Column 2, Column 3, etc`). (The column names can be changed by the user in the user interface.)
+
+## File source
+A file on your local hard drive or a file stream on a URL with http or https.
+
+## How to import data into Lumen
+CSV data is imported using `New->Dataset` in the Lumen user interface. Functionality will be added to refresh external data, both manually and on a schedule.
+
+## What you can't do right now
+
+#### Authentication
+
+Lumen does not yet support using external data sources that require authentication for the moment. This functionality is in the roadmap and will be available in some form.

```

Comment:
We authenticate/authorize Flow imports, but I'm not sure if that's relevant here.

In a sense we do support authentication via url encoded basic auth such as `https://Aladdin:OpenSesame@www.example.com/index.html`. Not sure if that needs to be mentioned here as it's not really a lumen feature. It's just an option for basic auth (instead of via headers)